components/grid/export/overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Overview
+page_title: Grid - Export Overview
+description: Export basics for the Grid for Blazor.
+slug: grid-export-overview
+tags: telerik, blazor, grid, export
+published: True
+position: 1
+---
+
+# Blazor Grid Export
+
+The Grid for Blazor provides a built-in functionality to export the data to:
+* [CSV](slug:grid-export-csv)
+* [Excel](slug:grid-export-excel)
+* [PDF](slug:grid-export-pdf)
+
+Before proceeding to the dedicated export articles, ensure you are familiar with the following specifics:
+
+## How the Export Works
+
+The Grid export feature has the following specifics:
+
+* If the Grid is using `OnRead` and is exporting all pages, it will fire an additional `OnRead` event at the time of exporting, with a request `PageSize` of `0`. This will enable the component to obtain all data.
+* The time for export will be longer if:
+    * The Grid has a lot of records and/or 
+    * The Grid is in a Client-side Blazor (WebAssembly) where all the code runs in the browser. At the time of writing, it is considerably slower than server-side Blazor, and it only has one actual thread.
```

Comment:
We used both terms in the dawn of Blazor, but I don't think this is necessary anymore, unless you want some specific SEO. No need for "considerably slower", the customers can decide for themselves.

```suggestion
    * The Grid is in a WebAssembly app where all the code runs in the browser and in one thread.
```